pkg/controllers/user/pipeline/remote/github/github.go

```diff
@@ -104,13 +104,7 @@ func (c *client) CreateHook(pipeline *v3.Pipeline, accessToken string) (string,
 	if err != nil {
 		return "", err
 	}
-	events := []string{}
-	if pipeline.Spec.TriggerWebhookPush {
-		events = append(events, "push")
-	}
-	if pipeline.Spec.TriggerWebhookPr {
-		events = append(events, "pull_request")
-	}
+	events := []string{"push", "pull_request"}
```

Comment:
why we create events for push and pull_request?

Reply:
We create github webhook with all needed events.
When it is triggered by any of these events. Hook handler check the pipeline config and choose to run it or ignore it.
In this way, we don't need to carefully inspect and update webhook config in github repo everytime the pipeline is updated(when some of the triggers can be enabled/disabled)